README.md

```diff
@@ -34,24 +168,17 @@ scrape_configs:
 
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'maia'
-    metrics_path: "/api/v1/metrics"
+    metrics_path: "/federate"
     basic_auth:
       # Corresponds to your OpenStack User and Project
-      username: <user_id>@<project_id>
+      username: <user_id>|<project_id>
```

Comment:
I am not convinced. Why be inconsistent with other uses of '@'? Also the vertical bar makes a nice optical separation of username and scope.

Just look briefly at

`<username>|<domain_name>@<project_name>|<domain_name>`

compare this to

`<username>@<domain_name>|<project_name>@<domain_name>`